src/lib/route.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+const routing = require('five-bells-routing')
+const ROUTE_EXPIRY = 45 * 1000 // milliseconds
+
+class Route extends routing.Route {
```

Comment:
When you extend a class, give it a name that explains how it's different. Looking at the code I'm actually not at all sure what it is changing vs the base class.

It also duplicates a lot of code, presumably because you need to override methods that explicitly reference the base class in the original code. This should be fixed by using `new this.constructor` or `new this` instead of `new Route`.